next.config.js

```diff
@@ -2,6 +2,9 @@
 
 module.exports = {
   reactStrictMode: true,
+  images: {
```

Comment:
What is this section for?

Reply:
`domains` configuration can be used to provide a list of allowed hostnames for external images.

Reply:
But won't it affect self-hosted instances?

Reply:
No longer needed. I checked the codebase, and we are not linking to any external images.